src/PointwiseFunctions/GeneralRelativity/GeneralizedHarmonic/CovariantDerivOfExtrinsicCurvature.hpp

```diff
@@ -0,0 +1,52 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/Tags.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+namespace gsl {
+template <typename>
+struct not_null;
+}  // namespace gsl
+class DataVector;
+/// \endcond
+
+namespace GeneralizedHarmonic {
+/*!
+ * \ingroup GeneralRelativityGroup
+ * \brief Computes the covariant derivative of extrinsic curvature
+ *
```

Comment:
Can you add the equations or give me a reference in a GitHub comment? I have no way of checking that what you've coded up matches some math :)

Reply:
Looking into it, this code is taking the partial derivative of the formula used in `PointwiseFunctions/GeneralRelativity/GeneralizedHarmonic/ExtrinsicCurvature.?pp`, plus using an identity for the partial derivative of the spacetime unit normal vector (I can't find the source of this identity after a quick search).

Reply:
Yup, updated the help text to include the derivation of what is coded up here.